tests/integration-tests/configs/released.yaml

```diff
@@ -170,11 +170,11 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["ubuntu2004"]
           schedulers: ["slurm"]
-    test_api.py::test_custom_image:
-      dimensions:
-        - regions: ["sa-east-1"]
-          instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["alinux2"]
+#    test_api.py::test_custom_image:
```

Comment:
Why are you commenting out this test?

Reply:
the commit descriptions have the details https://github.com/aws/aws-parallelcluster/pull/6327/commits